libs/gltfio/src/DracoCache.cpp

```diff
@@ -117,10 +117,10 @@ DracoMesh* DracoMesh::decode(const uint8_t* data, size_t dataSize) {
     return new DracoMesh(new DracoMeshDetails { std::move(meshStatus).value() });
 }
 
-void DracoMesh::getFaceIndices(cgltf_accessor* target) const {
+bool DracoMesh::getFaceIndices(cgltf_accessor* target) const {
     // Return early if we've already decompressed this data.
     if (target->buffer_view) {
-        return;
+        return true;
```

Comment:
is a precondition violated here? Or is it legitimate to have a null buffer_view?

Reply:
This is not a precondition violation, it just checks if the data has already been decompressed.